### PR TITLE
Multithreading

### PR DESCRIPTION
**Pull request by Louvtt**
- Multithreading of websocket + main app loop
- Adding login and connect functions
- Adding settings menu in chat
- Simu backend by Foxy for testing (to remove)
- Message handling
- More locale vars.

### CODE REVIEW

**Comment by FirefoxyleGibus** [comment]
seems good to me

File: src/termutil.py [hunk 1]
```diff
@@ -1,116 +1,111 @@
...
-				renderedText += "".join([chr(int(prng(beat, k))%(0x7e-0x20) + 0x20) for k in range(len(i.replace("￼ø", "{").replace("ŧ￼", "}")))])
-			else:
-				renderedText += i.replace("￼ø", "{").replace("ŧ￼", "}")
-		else:
-			if glitchifyNext:
-				renderedText += "".join([chr(int(prng(beat, k))%(0x7e-0x20) + 0x20) for k in range(len(i.replace("￼ø", "{").replace("ŧ￼", "}")))])
-			else:
-				renderedText += i.replace("￼ø", "{").replace("ŧ￼", "}")
-	return renderedText
+    text = text.replace(r"\{", "￼ø").replace(r"\}", "ŧ￼").replace("{", "�{").replace("}", "}�")
```
Comment (HastagGuigui) [inline src/termutil.py]: ok but why raw-string the areas where you actually want backslashes before the bracket so it can be actually displayed

**Review by HastagGuigui — CHANGES_REQUESTED** [review]
yeah there's not a lot of stuff to debate except that one weird questionable change in src/termutil.py:28, ill do it right away hang on

**Review by HastagGuigui — APPROVED** [review]
ok fixed

**Pull request merged**